internal/statfs_test.go

```diff
@@ -0,0 +1,16 @@
+package internal
+
+import "testing"
+
+func TestFSType(t *testing.T) {
```

Comment:
This test doesn't seem particularly useful, as in in doesn't test getTracefsPath. How about checking that if err != nil, neither paths exist?

Reply:
I can remove the test. I was mostly using it as local validation that the code returned what was expected.